package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "webpack-demo",
+  "name": "webpack",
   "version": "1.0.0",
-  "description": "",
+  "description": "<a name=\"readme-top\"></a>",
   "private": true,
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
- [ ] Kindly make sure that Webpack dev server is set up and running with no error, So kindly add the following script to easily run the dev server as well:

```
"scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
     "start": "webpack serve --open",
    "build": "webpack"
  },
```